packages/host/app/components/matrix/room.gts

```diff
@@ -173,12 +177,14 @@ export default class Room extends Component<RoomArgs> {
           @disabled={{this.doChooseCard.isRunning}}
           {{on 'click' this.chooseCard}}
         >Choose Card</Button>
-        <Button
-          data-test-send-open-cards-btn
-          @loading={{this.doSendMessage.isRunning}}
-          @disabled={{this.doSendMessage.isRunning}}
-          {{on 'click' this.sendOpenCards}}
-        >Send open cards</Button>
+
+        <label for='share-checkbox'>Share top cards</label>
```

Comment:
do you mean? "Share cards in stack"?

Reply:
Then again, I am not sure if the user will know what it even does? Im not too sure what results I am expecting from this PR

Reply:
Yeah we don't have design yet for this, I'm happy to make any changes to make it clearer before then.

Reply:
> do you mean? "Share cards in stack"?

This shares only the cards at the top of the stack, I'll change the wording there as it's not clear

Reply:
I've made this probably overly specific, but hopefully clearer for development & intent until there's a full design.